sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/api/Azure.ResourceManager.LiftrQumulo.netstandard2.0.cs

```diff
@@ -0,0 +1,480 @@
+        public virtual Azure.Response<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource> Get(string jobDefinitionName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource>> GetAsync(string jobDefinitionName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.LiftrQumulo.JobDefinitionResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class JobDefinitionData : Azure.ResourceManager.Models.ResourceData
+    {
+        public JobDefinitionData(Azure.ResourceManager.LiftrQumulo.Models.CopyMode copyMode, string sourceName, string targetName) { }
+        public string AgentName { get { throw null; } set { } }
+        public string AgentResourceId { get { throw null; } }
+        public Azure.ResourceManager.LiftrQumulo.Models.CopyMode CopyMode { get { throw null; } set { } }
+        public string Description { get { throw null; } set { } }
+        public string LatestJobRunName { get { throw null; } }
+        public string LatestJobRunResourceId { get { throw null; } }
+        public Azure.ResourceManager.LiftrQumulo.Models.JobRunStatus? LatestJobRunStatus { get { throw null; } }
+        public Azure.ResourceManager.LiftrQumulo.Models.ProvisioningState? ProvisioningState { get { throw null; } }
+        public string SourceName { get { throw null; } set { } }
+        public string SourceResourceId { get { throw null; } }
+        public string SourceSubpath { get { throw null; } set { } }
+        public string TargetName { get { throw null; } set { } }
+        public string TargetResourceId { get { throw null; } }
```

Comment:
Is this a `arm-id`? If yes please correct it by following https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/polishing.md#change-the-format-of-a-property